pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -319,24 +317,38 @@ def __init__(self, param, depth):
         self.optsChanged(self.param, self.param.opts)
 
     def updateDepth(self, depth):
-        ## Change item's appearance based on its depth in the tree
-        ## This allows highest-level groups to be displayed more prominently.
-        if depth == 0:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(100,100,100)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(220,220,255)))
-                font = self.font(c)
-                font.setBold(True)
-                font.setPointSize(font.pointSize()+1)
-                self.setFont(c, font)
+        """
+        Change set the item font to bold and increase the font size on outermost groups.
+        """
+        app = QtWidgets.QApplication.instance()
+        palette = app.palette()
+        color = palette.window().color()
+        h, s, l = color.hue(), color.saturation(), color.lightness()
+
+        stylesheet =  app.styleSheet()
+        textColor = None
+        if '/* Light Style' in stylesheet or '/* Dark Style' in stylesheet:
+            background = QtGui.QColor(255,255,255,0)
+            altBackground = background
+            textColor = QtGui.QColor('#000000')
+            if '/* Light Style' in stylesheet:
+                textColor = QtGui.QColor('#F0F0F0')
+        elif app.property('darkMode'):
+            background = QtGui.QColor.fromHsl(h, s, l * .6, 255)
```

Comment:
It might be better to work with the float version of the Hsl methods, should be `fromHslF()` here.

I would also suggest to stick to the original palette background here and change as few colors as possible.